crates/core/src/nats/streams.rs

```diff
@@ -0,0 +1,176 @@
+use std::collections::HashMap;
```

Comment:
As discussed, we need a middle ground for the Streams and Subjects. That'll be in my next PR.

Reply:
✅